.travis.yml

```diff
@@ -34,8 +31,7 @@ os:
   - linux
 notifications:
   email:
-    - dedukti-commits@lists.gforge.inria.fr
+    - genestier@lsv.fr
```

Comment:
Keep dedukti-commits please.

Reply:
I will of course keep dedukti-commit, but for now I'm doing a few experiments on this branch and I'm quite sure you don't want to receive a mail each time I do a non-working commit.
When it will be more stable, I will change it.